api/constants/constants.go

```diff
@@ -384,6 +384,14 @@ const (
 	// TraitGCPServiceAccounts is the name of the role variable used to store
 	// allowed GCP service accounts.
 	TraitGCPServiceAccounts = "gcp_service_accounts"
+
+	// TraitHostUserUID is the name of the variable used to specify
+	// the uid to create host user account with.
+	TraitHostUserUID = "teleport.dev/uid"
```

Comment:
You've tested this with Okta, right? Does it work okay with the attribute name that contains forward slash?

I wonder if we should change it to just `host_user_uid` and `host_user_gid`, to also be consistent with other trait names.

Reply:
Yeah it works okay, i think it does look a bit weird to have the slash, i'll change it to host_user_*